stistools/defringe/mkfringeflat.py

```diff
@@ -272,7 +281,8 @@ def mkfringeflat(inspec, inflat, outflat, do_shift=True, beg_shift=-0.5, end_shi
         # Apply the best shift and create output array
         flt_blk = block_reduce(fltdata, (binlines/fltbinlines, bincols/fltbincols),
                                 func=np.mean)
-        shifted_flat = shift(flt_blk, theshift, order=1, mode='nearest')
+        shifted_flat = shift(flt_blk[shiftrowstart:shiftrowstop, shiftcolstart:shiftcolstop],
```

Comment:
Python arrays are typically `arr[col, row]`.  Is this correct here?

Reply:
No, it's the other way round.  Python is [row, column], IRAF is [column, row]

Reply:
Yes, of course.  Comments after 5...